### PR TITLE
Support write,chmod,chown,rename in StackFS

### PR DESCRIPTION
**Pull request by LuQQiu**
Support StackFS operations with alluxio-fuse script
```
$ integration/fuse/bin/alluxio-fuse mount -s fuseMount fuseBeMounted
Starting StackFS process: mounting src local path "fuseBeMounted" to dst local path "fuseMount" with options="-obig_writes"
$ integration/fuse/bin/alluxio-fuse stat
pid	mount_point	source_path
94231	fuseMount	fuseBeMounted
$ integration/fuse/bin/alluxio-fuse umount fuseMount
Unmount fuse at fuseMount (PID: 94231).
```
Support write related operations in StackFS.

### CODE REVIEW

**Comment by LuQQiu** [comment]
Error: 6.937 [ERROR] Low: Check for sign of bitwise operation in alluxio.util.io.FileUtils.translateModeToPosixPermissions(int) [alluxio.util.io.FileUtils] At FileUtils.java:[line 164] BIT_SIGNED_CHECK
Error: 6.938 [ERROR] Low: Check for sign of bitwise operation in alluxio.util.io.FileUtils.translateModeToPosixPermissions(int) [alluxio.util.io.FileUtils] At FileUtils.java:[line 151] BIT_SIGNED_CHECK

File: core/common/src/main/java/alluxio/util/io/FileUtils.java [hunk 1]
```diff
@@ -134,6 +135,51 @@ public static short translatePosixPermissionToMode(Set<PosixFilePermission> perm
...
+   * @param mode the file mode
+   * @return posix file permissions
+   */
+  public static Set<PosixFilePermission> translateModeToPosixPermissions(int mode)
+      throws IOException {
+    Set<PosixFilePermission> perms = new HashSet<>();
+    // add owners permission
+    if (mode < 0) {
+      throw new IOException("Mode can not be a negative value");
+    }
```
Comment (apc999) [inline core/common/src/main/java/alluxio/util/io/FileUtils.java:150]: This should not throw IOE:
```suggestion
    Preconditions.checkArgument(mode >=0, "Mode can not be a negative value");
```

File: core/common/src/main/java/alluxio/util/io/FileUtils.java [hunk 2]
```diff
@@ -134,6 +135,49 @@ public static short translatePosixPermissionToMode(Set<PosixFilePermission> perm
...
   }
 
+  /**
+   * Translate mode to posix file permissions.
+   *
+   * @param mode the file mode
+   * @return posix file permissions
+   */
+  public static Set<PosixFilePermission> translateModeToPosixPermissions(int mode)
+      throws IOException {
```
Comment (apc999) [inline core/common/src/main/java/alluxio/util/io/FileUtils.java]: remove IOE

File: core/common/src/main/java/alluxio/util/io/FileUtils.java [hunk 3]
```diff
@@ -134,6 +135,49 @@ public static short translatePosixPermissionToMode(Set<PosixFilePermission> perm
...
+    Set<PosixFilePermission> perms = new HashSet<>();
+    // add owners permission
+    Preconditions.checkArgument(mode >=0, "Mode can not be a negative value");
```
Comment (apc999) [inline core/common/src/main/java/alluxio/util/io/FileUtils.java]: space before 0

**Review by apc999 — APPROVED** [review]

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Pull request merged**